Description: Fix import of django.conf.urls
 In the current source, we have:
  from django.conf.urls.defaults import patterns, include, url
 which is deprecated in Django 1.4, and not supported in 1.6. This patch
 removes the ".defaults" bit.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-12-24

--- python-django-openstack-auth-1.1.3.orig/openstack_auth/urls.py
+++ python-django-openstack-auth-1.1.3/openstack_auth/urls.py
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from django.conf.urls.defaults import patterns, url
+from django.conf.urls import patterns, url
 
 from .utils import patch_middleware_get_user
 
